Source/VFECore/Abilities/AbilityDef.cs

```diff
@@ -28,22 +28,28 @@ public class AbilityDef : Def
         public float              range            = 0f;
         public float              minRange         = -1f;
         public List<StatModifier> rangeStatFactors = new List<StatModifier>();
+        public List<StatModifier> rangeStatOffsets = new List<StatModifier>();
 
         public float              radius            = 0f;
         public float              minRadius         = -1f;
         public List<StatModifier> radiusStatFactors = new List<StatModifier>();
+        public List<StatModifier> radiusStatOffsets  = new List<StatModifier>();
 
         public float              power            = 0f;
         public List<StatModifier> powerStatFactors = new List<StatModifier>();
+        public List<StatModifier> powerStatOffsets = new List<StatModifier>();
 
         public int                castTime            = 0;
         public List<StatModifier> castTimeStatFactors = new List<StatModifier>();
+        public List<StatModifier> castTimeStatOffsets    = new List<StatModifier>();
```

Comment:
```suggestion
        public List<StatModifier> castTimeStatOffsets = new List<StatModifier>();
```
Please be consistent with your spacing.